package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/sinon": "^9.0.4",
```

Comment:
shouldn't it be in devDependencies. You don't need it tho'. Just nuke sinon completely

Reply:
Alright sir